### PR TITLE
Autonity network

### LINKED ISSUE #5: [autonity-network] Support Helm 3
### User story
As a user
I want to use Helm 3 for deployments
So that I can deploy charts using Helm 3

### Description
For a [full list of changes](https://helm.sh/blog/helm-3-released/). For the [migration guide](https://helm.sh/docs/topics/v2_v3_migration/).

### Acceptance criteria
- [x] Set `apiVersion: 1` in `Chart.yaml`
- [x] Ensure deployment is still successful
- [x] Updated documentation

### LINKED ISSUE #17: Add the $GITHUB_ACTOR variable as the name space during deployments and tests
As a user
I want to be able to identify my tests in the deployed cluster
So that I can see all of the logs

### Description
Not only is deploying to the `default` namespace bad practice, it makes identifying builds harder, and can also potentially cause conflicts when multiple people are running the GitHub actions on the same time.

### Acceptance criteria
- [x] Ensure namespace is user specific, i.e. using `$GITHUB_ACTOR`
- [x] Ensure namespace has unique number, i.e. using `${{ github.run_number }}`

### PR DESCRIPTION
**Pull request by Klazomenai**
### Changelog
Resolves #5 
Resolves #17 

### All Submissions

* [x] Have you followed the guidelines in our [Contributing document](CONTRIBUTING.md)?
* [x] Is the `branch` name the same as Chart name that you will release?
* [x] Is your changes touch only one chart?
* [x] Is the version of chart incremented in `Chart.yaml`?
* [x] Is the [Change Log document](CHANGELOG.md) updated ?

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -51,12 +51,15 @@ jobs:
 
     - name: Deploy the chart
       run: |
-        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace default
+        kubectl create namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
```
Comment (eastata) [inline .github/workflows/pr.yaml]: It return error if namespace already exist

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -51,12 +51,15 @@ jobs:
...
+        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
         # Sadly --wait fails, so have to sleep, look to maybe pull in mock test to solve
         # this problem another way
         sleep 60
     - name: Run the tests
-      run: helm test $GITHUB_HEAD_REF-$SHORT_SHA
+      run: helm test $GITHUB_HEAD_REF-$SHORT_SHA --namespace $ACTOR_LOWERCASE-$GITHUB_RUN_ID
```
Comment (eastata) [inline .github/workflows/pr.yaml:66]: If we run at the same time PR check and branch, or tag based on the same git commit?
is it have sense to replace chart name from `$GITHUB_HEAD_REF-$SHORT_SHA` to `$ACTOR_LOWERCASE-$GITHUB_RUN_ID` for example?

File: stable/autonity-network/README.md [hunk 3]
```diff
@@ -2,130 +2,123 @@
...
+### Prerequisites
+1. Install [Kubernetes](http://kubernetes.io) `v1.18.4`. Basic features have been tested in both Minikube and Kind with extra functionality available in EKS and GKE:
+   - Local: [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+   - Local: [Kind](https://kind.sigs.k8s.io/docs/user/quick-start)
+   - Cloud: [Amazon EKS](https://eksworkshop.com/prerequisites/self_paced/)
+   - Cloud: [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+1. Install Helm [3.2.4](https://github.com/helm/helm/releases/tag/v3.2.4)
+1. Initialise [the official](https://helm.sh/docs/intro/quickstart/ the official) `@stable` and the Autonity Helm charts repositories:
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```
Comment (eastata) [inline stable/autonity-network/README.md:20]: Why are we need it?

**Review by eastata — APPROVED** [review]

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (Klazomenai) [reply .github/workflows/pr.yaml:66]: `$GITHUB_RUN_ID ` is used in the namespace if you need it for identification.

File: stable/autonity-network/README.md (hunk 3 above)
Comment (Klazomenai) [reply stable/autonity-network/README.md:20]: [No repositories are shipped with Helm 3.](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/)

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (eastata) [reply .github/workflows/pr.yaml:66]: I need to avoid error when we have concurrent jobs with same chart names

**Pull request merged**